src/GoTrueClient.ts

```diff
@@ -135,7 +136,8 @@ export default class GoTrueClient {
   }> {
     try {
       this._removeSession()
-
+      Logger.warn('redirectTo param will only redirect OAuth logins.')
```

Comment:
not sure why we added a `redirect_to` to the `signUp` method - it won't work here as well since these are all POST requests